GNUmakefile

```diff
@@ -33,7 +33,7 @@ python_sample_dir = sample
 js_output_dir = js_out
 js_runtime_dir = javascript
 js_sample_dir = sample
-NODE ?= nodejs
+NODE ?= node
```

Comment:
Maybe a worthwhile change, but seems unconnected with added ES6 support.

Reply:
I've changed this in master (and renamed the variable to `JSRUN` since it ought to be possible to get this to work with e.g. deno).